guac_core/Cargo.toml

```diff
@@ -32,6 +31,7 @@ hex = "0.3.2"
 diesel = { version = "=1.2", features = ["sqlite"] }
 dotenv = "0.10"
 qutex = "^0.2"
+owning_ref = {git="https://github.com/kingoflolz/owning-ref-rs/", branch="fix-bug"}
```

Comment:
I assume you've got a PR open on the stuff they lacked?

Reply:
yup, its been open for like a couple months at this point lol https://github.com/Kimundi/owning-ref-rs/pull/44